src/chordChart.ts

```diff
@@ -786,22 +792,24 @@ module powerbi.extensibility.visual {
                         tooltipInfo.push(ChordChart.createTooltipInfo(
                             this.data.labelDataPoints,
                             this.data.dataMatrix,
-                            tooltipEvent.data.source));
+                            tooltipEvent.data.source,
+                            this.localManager));
 
                         tooltipInfo.push(ChordChart.createTooltipInfo(
                             this.data.labelDataPoints,
                             this.data.dataMatrix,
-                            tooltipEvent.data.target));
+                            tooltipEvent.data.target,
+                            this.localManager));
                     }
 
                     return tooltipInfo;
                 });
         }
 
-        private static createTooltipInfo(labelDataPoints: ChordArcDescriptor[], dataMatrix: number[][], source: any) {
+        private static createTooltipInfo(labelDataPoints: ChordArcDescriptor[], dataMatrix: number[][], source: any, localManager: ILocalizationManager) {
```

Comment:
Looks like `localManager` is not used here.

Can we remove it from arguments?

Reply:
Sure, my mistake. I will fix